test/integration/auto-export/test/index.test.js

```diff
@@ -91,13 +91,15 @@ describe('Auto Export', () => {
     })
 
     it('should include error link when hydration error does occur', async () => {
-      const browser = await webdriver(appPort, '/post-1/hydrate-error')
+      const browser = await webdriver(appPort, '/post-1/hydrate-error', {
+        pushErrorAsConsoleLog: true,
+      })
       const logs = await browser.log()
       expect(logs).toEqual(
         expect.arrayContaining([
           {
             message: expect.stringContaining(
-              'See more info here: https://nextjs.org/docs/messages/react-hydration-error'
+              'https://react.dev/link/hydration-mismatch'
```

Comment:
After this we need to rewrite the error overlay feature. There are too many places where we intercept error events and console.error. Some seems unused now, some duplicate.

In Pages router, hydration errors are already unhandled runtime errors which is bad. In the rewrite, it should be aparent what belongs to Pages router (any React version) and what belongs to App router (a single React version).